src/storage/Storage.ts

```diff
@@ -11,6 +11,7 @@ export type SyncChangeListener = (attempt: StoredAttempt[]) => void;
 
 export declare interface Storage {
   connectRemoteDB(username: string, password: string): void;
```

Comment:
Would you mind making the signature like this instead?
```suggestion
  connectRemoteDB(username: string, password: string, options?: {databaseURL: string}): void;
```